src/modules/journal/journal.selector.ts

```diff
@@ -28,6 +27,23 @@ export const canNavigateToPreviousDay = (journal: JournalModel, today: DateTime)
   return selectedDate > lookbackLimit;
 };
 
+/**
+ * Function to check if there is data after selected date in the journal slots, and to
+ * return a true signifying that the user can navigate back a day if that is the case.
+ * @param {JournalModel} journal
+ * @return {boolean}
+ */
+export const hasSlotsAfterSelectedDate = (journal: JournalModel): boolean => {
```

Comment:
As our convention goes: we try to write clean, understandable code and not comments around our functions. The code should be the documentation. You selected a very clear and understandable function name, input names, and we have the return type visible. That is enough for us to understand what this function does.

In the implementation, try to be more verbose and make sure that everything makes sense for the person who reads the code. 

If you do these things then you don't need to write comments, because it's just a maintenance overhead and weirdly elevate the complexity of the code.